### PR TITLE
added missing test for habitat backend

### PR DESCRIPTION
**Pull request by Dhiraj100892**
## Motivation and Context
- Added missing tests for `habitat` backend 
- Removed `initial_rotation` param from `get_current_ pcd` and `pix_to_3dpt` from habitat camera backend

## How Has This Been Tested

Using `pytest`

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/test_habitat_locobot.py [hunk 1]
```diff
@@ -26,27 +28,167 @@ def create_robot():
...
 )
 def test_camera_position_control(create_robot, target_position):
+    thr = 1e-3
     bot = create_robot
     bot.camera.reset()
     bot.camera.set_pan_tilt(target_position[0], target_position[1], wait=True)
-    achieved_position = bot.camera.get_state()
-    ag_error = np.fabs(np.array(achieved_position) - np.array(target_position))
-    assert np.max(ag_error) < 0.10
+    assert np.sum(np.abs(np.array(bot.camera.get_state()) - np.array(target_position))) < thr
```
Comment (soumith) [inline tests/test_habitat_locobot.py:45]: you can also use something like `np.allclose` if it's better: https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

**Review by soumith — APPROVED** [review]
good stuff. lgtm

File: tests/test_habitat_locobot.py (hunk 1 above)
Comment (Dhiraj100892) [reply tests/test_habitat_locobot.py]: @soumith thanks for recommending `np.allclose`. I think its more suitable. I have updated the PR based on it

Comment (soumith) [reply tests/test_habitat_locobot.py]: great, the changes look good

**Pull request merged**